src/FMI2.jl

```diff
@@ -395,7 +395,7 @@ function fmi2Load(pathToFMU::String; unpackPath=nothing)
     end
 
     # make URI ressource location
-    tmpResourceLocation = string("file:/", fmu.path)
+    tmpResourceLocation = string("file:///", fmu.path)
```

Comment:
isn't that windows specific? Using `joinpath` would probably be safer IIRC. I don't know, @oxinabox is the file guy.

Reply:
Perhaps yeah. I wanted to see if it broke the CI tests.